### DISCUSSION TITLE
build works on macOS but not on Windows - what is the issue?

### DISCUSSION THREAD

**Discussion by clstaudt** [pyinstaller/pyinstaller#7171] [Help] [answered]
I can build the app (https://github.com/tuttle-dev/tuttle) for macOS using pyinstaller. Windows is another story - looking for some help here to find out what the issue is.

I have a GitHub Action building the app for Windows. The action calls:

`pyinstaller --noconfirm --windowed --clean --onefile --add-data=tuttle_tests/data;./tuttle_tests/data --add-data=templates;./templates`

 It runs without throwing an error: https://github.com/tuttle-dev/tuttle/actions/runs/3275331867/jobs/5390040493

The result is a 45 MB .exe file. Suspiciously small compared to the .app bundle for macOS (ca. 300MB).

When I run the .exe, I get the following:

````python
Traceback (most recent call last):
  File "Tuttle.py", line 32, in <module>
  File "PyInstaller\loader\pyimod02_importers.py", line 499, in exec_module
  File "views.py", line 25, in <module>
ModuleNotFoundError: No module named 'tuttle'
````

That means the Python environment is misconfigured, with the app's own core Python module missing.

**Comment by rokm** [comment]
I would guess that an entry point named `Tuttle.py` will end up shadowing the `tuttle` package during analysis on a case-insensitive OS? Can you try renaming it to something else?

**Reply by clstaudt** [reply]
@rokm  Tried it. That is not the issue, the error remains the same.

**Comment by clstaudt** [comment, marked as answer]
I found the issue: The GitHub action for the build did not install the library itself. Sorry, nothing to do with pyinstaller - please close the thread.
